### PR TITLE
improve multiexposure impact function and add summary about analysis layer

### PR DESCRIPTION
**Pull request by Gustry**
### What does it fix?
* Ticket: #4439
* Funded by: DFAT
* Description: improve the multi-exposure impact function:
  * Manage datastore
  * Merge analysis layer

This PR is introducing fields with 2 parameters to build the field name and the key @myarjunar. Before it's maximum one.

I didn't commit the aggregation summary, it's empty in the code. I will do that in another PR.

### Checklist:
- [ ] Commit messages are descriptive and explain the rationale for changes
- [ ] Add to the changelog in metadata.txt if it's a new feature
- [ ] Unit test for new code added
- [ ] Request someone to review or test your PR

### CODE REVIEW

File: safe/gis/vector/summary_5_multi_exposure.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+    read_dynamic_inasafe_field,
+)
+
+__copyright__ = "Copyright 2017, The InaSAFE Project"
+__license__ = "GPL version 3"
+__email__ = "info@inasafe.org"
+__revision__ = '$Format:%H$'
+
+
+def multi_exposure_analysis_summary(analysis, intermediate_analysis):
```
Comment (ismailsunni) [inline safe/gis/vector/summary_5_multi_exposure.py]: I saw in this function you merge the analysis layers into one. What about the metadata? Especially the provenance

Comment (Gustry) [reply safe/gis/vector/summary_5_multi_exposure.py]: I'm just merging QGIS layers. The same as summary_X.py in the gis/vector package.
We still need to know/do about provenance.

**Comment by Gustry** [comment]
So can I merge it @ismailsunni?

**Comment by Gustry** [comment]
Travis can't download the docker image in my PR, 3 times I started it 3 times. But it could work for @ismailsunni on the develop branch this morning

**Comment by myarjunar** [comment]
Thanks @Gustry !

**Pull request merged**